zulip_bots/zulip_bots/lib.py

```diff
@@ -156,7 +158,58 @@ def run_message_handler_for_bot(lib_module, quiet, config_file, bot_name):
 
     state_handler = StateHandler()
 
+    # Bot details and default commands from defaults, then override if provided
+    bot_details = { 'name': bot_name.capitalize(),
+                    'description': "",
+                    'commands': {},
+                    'no_defaults': False,
+    }
+    def def_about():
+        desc = bot_details['description']
+        return "**{}**{}".format(bot_details['name'],
+                                 "" if desc == "" else ": {}".format(desc))
+    def def_help():
+        return ("\n".join("**{}** - {}".format(k, v[1])
+                          for k, v in default_commands.items() if k != '') +
+                "\n" +
+                "\n".join("**{}** - {}".format(k, v)
+                          for k, v in bot_details['commands'].items() if k != ''))
+    def def_commands():
+        return "**Commands**: {} {}".format(
+            " ".join(k for k in default_commands if k != ''),
+            " ".join(k for k in bot_details['commands'] if k != ''))
+    default_commands = OrderedDict([
```

Comment:
If these are not ordered, then when iterated in the commands and help text, the order is not guaranteed - perhaps not *vital* for regular use, but tests won't be consistent either!